game_eggs/steamcmd_servers/arma/README.md

```diff
@@ -2,5 +2,6 @@
 
 ARMA is a series of first-person tactical military shooters, originally released for Microsoft Windows. It features large elements of realism and simulation; a blend of large-scale military conflict spread across large areas alongside the more close quartered battles.
 
-## Arma 3
+## Arma Titles
+* Arma 2 (Egg Requested)
```

Comment:
It can be included in the readme when such an egg exists. At this moment, that's unnecessary.

Reply:
Agreed. Fixed 33e0e46